examples/powercli/nsx-t_tag-sync/stack.yml

```diff
@@ -0,0 +1,16 @@
+provider:
+  name: openfaas
+  gateway: https://phxlvveba01.itplab.local
+functions:
+  nsxttagsync:
+    lang: powercli
+    handler: ./handler
+    image: cmstraka/veba-powercli-tagging:latest
```

Comment:
The guidance for a new function is to have its own container image (e.g. do not re-use an existing) and it would contain the handler logic for your use case (e.g. sync vSphere Tags to NSX-T Tags). This way, once verified, we will build an official image and users simply just need to update the `nsx-secrets.json` to be able to use this function w/o  having to build

Maybe something like `veba-powercli-tag-sync-to-nsxt`

Reply:
Done.  Thanks for the feedback!